docs/webpack.static.render.config.js

```diff
@@ -27,34 +29,56 @@ const config = {
   },
 
   module: {
-    loaders: [
+    rules: [
       {
         test: /\.js$/,
-        loader: 'babel',
-        query: babelConfig,
-        include: appPaths
+        include: appPaths,
+        use: {
+          loader: 'babel-loader',
+          options: babelConfig
+        }
       },
       {
         test: /\.less$/,
-        loader: 'css/locals?modules&localIdentName=[name]__[local]___[hash:base64:5]!postcss!less',
-        include: appPaths
+        include: appPaths,
+        use: [
+          {
+            loader: 'css-loader/locals',
+            options: {
+              modules: true,
+              localIdentName: '[name]__[local]___[hash:base64:5]'
+            }
+          },
+          {
+            loader: 'postcss-loader',
+            options: {
+              plugins: [autoprefixer]
```

Comment:
As mentioned [here](https://github.com/seek-oss/seek-style-guide/pull/341/files#r143368472)

Reply:
This one isn't necessary because we're not actually generating a style sheet in the static render pass—it's just to generate the localised class names.